modules/tinymce/CHANGELOG.md

```diff
@@ -8,11 +8,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Added a new `toolbar_sticky_offset` setting to allow the toolbar to be offset from the top of the page #TINY-7337
+- Added a new `mceFocus` command that focuses the editor. Equivalent to using `editor.focus()` #TINY-7373
 
 ### Changed
 - Changed the load order so that the content css gets loaded before the editor gets populated with contents. #TINY-7249
 
 ### Fixed
+- The editor content could be edited after calling `setProgressState(true)` #TINY-7373
```

Comment:
Do we need to mention that this fix only applies to iframe mode?

Reply:
Good point. Have added `...in iframe mode` to the end but not quite sure if that is clear enough